post-scan-actions/aws-nodejs-securityhub-integration/serverless-app-template.yml

```diff
@@ -0,0 +1,41 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+
+Parameters:
+  ScanResultTopicARN:
+    Type: String
+    Description: The ARN of the scan result SNS topic in storage stack.
+  AWSACCOUNTNO:
+    Type: Number
+    Description: Provide AWS Account ID.
+  AWSSecurityHubARN:
+    Type: String
+    Description: Provide AWS Security hub ARN from selected region
+
+Resources:
+  C1FSecurityHub:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: s3://bucket-4716/lambda.zip
```

Comment:
I don't think it's a good idea to add a new template and new code from unmanaged bucket. Instead, modify the [template.yaml](https://github.com/trendmicro/cloudone-filestorage-plugins/blob/master/post-scan-actions/aws-nodejs-securityhub-integration/template.yaml) would with a Metadata section like [this](https://github.com/trendmicro/cloudone-filestorage-plugins/blob/master/post-scan-actions/aws-python-promote-or-quarantine/template.yml#L4) would make more sense.